CMakeLists.txt

```diff
@@ -780,6 +790,15 @@ add_library (seastar
   src/websocket/server.cc
   )
 
+# Shared library soname  
+set_target_properties(seastar PROPERTIES
+  VERSION ${PROJECT_VERSION}.${Seastar_API_LEVEL}.${Seastar_RELEASE_DATE}
+  SOVERSION ${PROJECT_VERSION})
+
+# We disable _FORTIFY_SOURCE because it generates false positives with longjmp() (src/core/thread.cc)
+set_source_files_properties(src/core/thread.cc
+  PROPERTIES COMPILE_FLAGS -U_FORTIFY_SOURCE)
```

Comment:
you are partially reverting c96444fbe43acac0d31418b65b04395c11fdda14. probably want to drop this change?